pages/common/fuck.md

```diff
@@ -10,3 +10,7 @@
 - Try to match a rule for the previous command:
 
 `fuck`
+
+- Try to match a rule for the previous command and confirm the first choice immediately:
```

Comment:
```suggestion
- Confirm the first choice immediately (correct argument only depends on level of frustration):
```
It's not immediately obvious what's the difference of the three options.

Reply:
They all do the same thing. I wish it was ` fuck --fuck` would be better.

its just a -y flag, infact -y is a flag you can use.

Reply:
```suggestion
- Confirm the first choice immediately (correct argument depends on level of annoyance):
```

Reply:
I'm fine with both